website/docs/d/db_instance.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
   Get information on an RDS Database Instance.
 ---
 
-# aws_db_instance
+# Data Source: aws_dbinstance
```

Comment:
Not sure if this was intentional, but it looks like some underscores were removed from a lot of these headers.

Reply:
argh that was not intentional